packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1147,6 +1147,32 @@ describe('shallow', () => {
       expect(wrapper.find('.b').find('.c')).to.have.lengthOf(6);
     });
 
+    it('can call find on the same wrapper more than once', () => {
```

Comment:
can we add the same test for `mount`?

Reply:
I've rebased this using the new method structure for tests, so we get dual coverage for free.